### PR TITLE
Added Color property to AKImageLoader

### PR DESCRIPTION
**Pull request by kengoon**
Added color property to AKImageLoader for dynamic change of color to match the application color

### CODE REVIEW

File: akivymd/uix/imageloader.py [hunk 1]
```diff
@@ -34,7 +34,8 @@ class AKImageLoader(ThemableBehavior, AsyncImage):
     bg_rec_opacity= NumericProperty(0)
     fr_rec_opacity= NumericProperty(0)
     circle= BooleanProperty(True)
-
+    bg_color= theme_cls.bg_dark
+    fr_color= theme_cls.bg_dark
```
Comment (quitegreensky) [inline akivymd/uix/imageloader.py]: these are properties then you must define them as properties

```
fr_color= ListProperty()
bg_color= ListProperty()
```

File: akivymd/uix/imageloader.py [hunk 2]
```diff
@@ -12,14 +12,14 @@
...
-            rgba: root.theme_cls.bg_darkest
+            rgba: root.fr_color
             a: root.fr_rec_opacity
         RoundedRectangle:
             pos: self.pos
             size: self.size
             radius: [(self.size[0]/2,self.size[1]/2) , (self.size[0]/2,self.size[1]/2),(self.size[0]/2,self.size[1]/2),(self.size[0]/2,self.size[1]/2) ] if root.circle==True else [(dp(20),dp(20)) , (dp(20),dp(20)) , (dp(20),dp(20)) , (dp(20),dp(20)) ]
         Color:
-            rgba: root.theme_cls.bg_dark   
+            rgba: root.bg_color
```
Comment (quitegreensky) [inline akivymd/uix/imageloader.py]: set default value from here 
`root.bg_color if root.bg_color else root.theme_cls.bg_dark `

File: akivymd/uix/imageloader.py [hunk 3]
```diff
@@ -12,14 +12,14 @@
     source: ' '
     canvas.before:
         Color:
-            rgba: root.theme_cls.bg_darkest
+            rgba: root.fr_color
```
Comment (quitegreensky) [inline akivymd/uix/imageloader.py]: `root.fr_color if root.fr_color else root.theme_cls.bg_darkest`

**Review by quitegreensky — CHANGES_REQUESTED** [review]
Hey. Solve this issues then squash commits into one

**Comment by kengoon** [comment]
I already made the changes and I don't understand what you mean by squashing the commit into one......(sorry I'm not very good at understanding some English terms)

**Comment by quitegreensky** [comment]
> 
> 
> I already made the changes and I don't understand what you mean by squashing the commit into one......(sorry I'm not very good at understanding some English terms)

Squashing is a git term. It merges two commits into one commit to prevent from unnecessary commits being merged with the target branch. currently you have two commits. [how to squash](https://www.internalpointers.com/post/squash-commits-into-one-git)

**Comment by kengoon** [comment]
i'm totally confused with all that, can you make that change your self please

**Pull request closed without merging**